frontend/src/metabase/components/form/CustomForm/CustomFormFooter/CustomFormFooter.styled.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+
+interface CustomFormFooterStyledProps {
+  shouldReverse?: boolean;
+}
+export const CustomFormFooterStyled = styled.div<CustomFormFooterStyledProps>`
+  display: flex;
+  align-items: center;
```

Comment:
Well... correctly the property name seems to make some forms have the button right in the middle as we're now changing the main flex axis to the vertical axis, so the `align-items` which aligns the flex items along the secondary axis (now horizontal axis), now place the item in the middle.

![image](https://user-images.githubusercontent.com/1937582/177179605-76f166c0-2112-4ef1-9da8-fa557ae4e4f1.png)

Could you at least make sure both:
- Slack form http://localhost:3000/admin/settings/slack and
- Login form

still works as expected?

Reply:
FYI @gusaiani